printer_mods/Gearnut/V0_Misumi_LED_Mount/README.md

```diff
@@ -0,0 +1,22 @@
+V0 Misumi Led mount
+
+![Alt_text](./Images/Cover.jpg)
+
+
+These were modified from Eddie's Misumi Led clips (https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/eddie/LED_Bar_Clip) as I like the design with them in my V2.4 and VT. They should fit most WS2812b Led strips you can find online though If it has the silicone cover on them you may need to peel them off or file down the inside track a touch.
```

Comment:
Can you make this a relative link too? (`../../printer_mods/eddie/LED_Bar_Clip`)

Reply:
Well figured that one out faster. its a relative link now as well